pkg/network/protocols/http/telemetry.go

```diff
@@ -96,8 +102,18 @@ func (t *Telemetry) Count(tx Transaction) {
 		t.hits5XX.Add(1)
 	}
 
-	if isEncrypted(tx) {
-		t.totalHitsEncrypted.Add(1)
+	switch tx.StaticTags() {
+	case GnuTLS:
+		t.totalHitsGnuTLS.Add(1)
+		return
```

Comment:
the return is redundant in every case, as go's default mechanism is NOT fallthrough